File: viajerosrepojava-master/viajerosrepojava-master/viajeros/viajeros/src/main/java/ps/com/viajeros/dtos/valuations/ValuationRequestDto.java
package ps.com.viajeros.dtos.valuations;

import jakarta.persistence.JoinColumn;
import jakarta.persistence.ManyToOne;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import ps.com.viajeros.entities.user.UserEntity;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class ValuationRequestDto {
    private Long idTrip;      // ID del viaje
    private Long idUserValuated;      // ID del usuario que es calificado (pasajero)
    private Long idUserWhoValuated;
    private Long rating;      // La calificación dada
    private String comments;  // Los comentarios adicionales, si es necesario
}

File: viajerosrepoangular-master/viajerosrepoangular-master/viajeros/src/app/models/Valuations/ValuationRequestDto.ts
export class ValuationRequestDto {
    idTrip: number;    // ID del viaje
    idUserValuated: number;    // ID del usuario que es calificado (pasajero)
    idUserWhoValuated: number;
    rating: number;    // La calificación dada
    comments: string;  // Los comentarios adicionales, si es necesario
  
    constructor(idTrip: number, idUserValuated: number, idUserWhoValuated:number, rating: number,  comments: string) {
      this.idTrip = idTrip;
      this.idUserValuated = idUserValuated;
      this.idUserWhoValuated = idUserWhoValuated;
      this.rating = rating;
      this.comments = comments;
    }
  }
  